packages/fast-development-site-react/src/components/toc/toc-menu.tsx

```diff
@@ -26,6 +26,15 @@ const dropdownInactive: string = "PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllci
 const style: ComponentStyles<ITocMenuManagedClasses, IDevSiteDesignSystem> = {
     toc__menu: {},
     toc__menu__button: {
+        fill: "none",
+        background: "none",
+        outline: "0",
+        border: "none",
+        position: "relative",
+        width: "100%",
+        fontFamily: "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif",
```

Comment:
we should set this at a top level and inherit everywhere else.

Reply:
That seems like a bug we should address.

Reply:
Its part of the user agent stylesheet in chrome... input, textarea, select, button have a font property set `font: 400 13.3333px Arial;`

Reply:
Yep - we should be including styles to reset in ours though. If not part of this, we should create a bug to fix this asap. You can also try `fontFamily: inherit`